hackathon_site/hardware/serializers.py

```diff
@@ -250,7 +250,8 @@ def validate(self, data):
     def create(self, validated_data):
 
         # make a copy of validated_data
-        validated_data_copy = validated_data.copy()
+        # validated_data_copy = validated_data.copy()
```

Comment:
Should't we remove this comment, since it's not in use

Reply:
Yea I was thinking the same thing! I think it will also be beneficial to comment that you removed the comment, in case one forgets in the future. It's always good to comment changes that become invisible like ghosts 👻